libkbfs/tlf_edit_history.go

```diff
@@ -528,6 +556,36 @@ func (teh *TlfEditHistory) updateHistory(ctx context.Context,
 	}
 
 	currEdits.addNewEdits(newEdits)
+	// Send the notifications.
+	for writer, edits := range newEdits {
+		for _, edit := range edits {
+			var n *keybase1.FSNotification
+			switch edit.Type {
+			case FileCreated:
+				cop, ok := edit.op.(*createOp)
+				if !ok {
+					return fmt.Errorf("No create op for create notification, "+
```

Comment:
Is this effectively acting like a `panic()` since it should never happen (we're throwing out all notifications in this case)?